src/main/groovy/io/seqera/wave/service/builder/BuildStrategy.groovy

```diff
@@ -37,6 +37,8 @@ abstract class BuildStrategy {
 
     abstract BuildResult build(BuildRequest req)
 
+    abstract String getLogs(String buildId)
```

Comment:
Ideally this should return an `InputStream`

Reply:
done